### ISSUE TITLE
RE: Changelog workflow

### ISSUE THREAD

**Issue by emmadesilva** [mallardduck/kickflip-monorepo#53]
Based on the readme todo item about sorting out workflow for `CHANGELOG.md`, thought I could share the workflow I set up for Hyde. https://github.com/hydephp/hyde/blob/master/.github/workflows/update-changelog.yml.

I think it looks pretty good! https://github.com/hydephp/hyde/blob/master/CHANGELOG.md

**Comment by mallardduck** [comment]
That does look really good! Seems like a great route for me to explore.

The only concern/though I have is around the repo subsplit - so I may need to implement my own workflow for it. That way I can run the auto-changelog tool in a way where it will update the "root" changelog as well as the "package" changelog.

**Comment by caendesilva** [comment]
Right, I'm not 100% clear on how the whole monorepo thing works yet but I'm sure you can figure it out :)
